tests/datatypes/test_bytes.py

```diff
@@ -105,23 +105,23 @@ class BinaryBytesOperationTests(BinaryOperationTestCase, TranspileTestCase):
     data_type = 'bytes'
 
     not_implemented = [
-        'test_add_bool',
+        # 'test_add_bool',
```

Comment:
I making the changes now, so I had a question: is there a way to resubmit a pull request after commiting changes or do I need to create a new pull request everytime?